NuGet.Config

```diff
@@ -1,8 +1,11 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <configuration>
   <packageSources>
+    <clear />
     <add key="nugetv3" value="https://api.nuget.org/v3/index.json" />
     <add key="myget" value="https://www.myget.org/F/azureiiot/api/v3/index.json" />
```

Comment:
I thought we are only allowed one? Marc removed the myget feed in Master...

Reply:
yep, just interim step, please approve Marcs PR, then we can close this one...